src/otx/core/data/entity/tile.py

```diff
@@ -252,3 +255,80 @@ def collate_fn(cls, batch_entities: list[TileInstSegDataEntity]) -> TileBatchIns
             masks=[tile_entity.ori_masks for tile_entity in batch_entities],
             polygons=[tile_entity.ori_polygons for tile_entity in batch_entities],
         )
+
+
+@dataclass
+class TileSegDataEntity(TileDataEntity):
+    """Data entity for segmentation tile task.
+
+    Attributes:
+        ori_masks (tv_tensors.Mask): The masks of the original image.
+    """
+
+    ori_masks: tv_tensors.Mask
+
+    @property
+    def task(self) -> OTXTaskType:
+        """OTX Task type definition."""
+        return OTXTaskType.SEMANTIC_SEGMENTATION
+
+
+@dataclass
+class TileBatchSegDataEntity(OTXTileBatchDataEntity):
```

Comment:
Could you add unit tests for this?

Reply:
It's covered already.
![image](https://github.com/user-attachments/assets/121cc43c-e1ab-4673-ae88-f2cd2472350c)